AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -13,6 +13,7 @@ struct ChooseSessionTypeView: View {
     @EnvironmentObject private var tabSelection: TabBarSelection
     @EnvironmentObject private var emptyDashboardButtonTapped: EmptyDashboardButtonTapped
     @EnvironmentObject private var finishAndSyncButtonTapped: FinishAndSyncButtonTapped
+    @EnvironmentObject private var exploreSessionsButton: ExploreSessionsButton
```

Comment:
I really don't like that we keep doing this... But I don't have an alternative right now. Gonna need to think about how to solve that. Maybe abstract routers one day...